pkg/cmd/openshift-controller-manager/configdefault/openshiftcontrolplane_default.go

```diff
@@ -0,0 +1,30 @@
+package configdefault
+
+import (
+	openshiftcontrolplanev1 "github.com/openshift/api/openshiftcontrolplane/v1"
+	"github.com/openshift/library-go/pkg/config/configdefaults"
+	leaderelectionconverter "github.com/openshift/library-go/pkg/config/leaderelection"
+)
+
+func SetRecommendedOpenShiftControllerConfigDefaults(config *openshiftcontrolplanev1.OpenShiftControllerManagerConfig) {
```

Comment:
should this move to library-go?

Reply:
(or openshift/api even)

Reply:
not in API (no methods). Probably not in library-go.  I think these are strictly the concern of this binary unlike defaults for the shared types.